analytics_export/bigqueryClient.js

```diff
@@ -110,6 +110,10 @@ class BigqueryClient {
     const table = this.dataset.table(`${tableName}$${date.replace(/-/g, "")}`);
 
     await this.loadSemaphore.use(async () => {
+      await table.query(
+        `DELETE FROM ${tableName} WHERE date = '${date}' AND app_name = '${appName}'`,
+      );
```

Comment:
It's worth a comment explaining why we can't just use `writeDisposition` as an argument to table.load here (I assume because that would truncate *all* the data, rather than just today's)

Reply:
It's because we only want to delete data for the current app so that writes don't create duplicate data.  All the apps are being written to the same table but in separate jobs.  `WRITE_TRUNCATE` does work for a single day as long as we add the `$YYYYMMDD` suffix to the table name.  I'll add a comment.